tabpy-tools/client_tests/test_rest.py

```diff
@@ -43,109 +41,109 @@ def mock_response(status_code):
 
     def test_GET(self):
         url = 'abc'
-        data = {'foo':'bar'}
+        data = {'foo': 'bar'}
         self.assertEqual(self.rnw.GET(url, data), 'json')
         self.rnw.session.get.assert_called_once_with(url,
-            params=data,
-            timeout=None)
+                                                     params=data,
+                                                     timeout=None)
```

Comment:
Any reason for `timeout=None`? Assuming this uses requests behind the scenes that is the default anyway so no need to explicitly state, though it's probably best to actually provide some type of value to prevent these from hanging infinitely

Reply:
Yeah... for the test code we will have more improvements sometime in future and for now, we just need to make sure existing tests whatever they are passing.